golang/_redirects

```diff
@@ -5,10 +5,8 @@
 #
 # To regenerate, run:
 #   go run ./tools/redir-gen/
-/actions-downstream-test/* go-get=1 /golang/actions-downstream-test.html 200
-/actions-kind/* go-get=1 /golang/actions-kind.html 200
```

Comment:
Removed, again didn't run the tool:

https://github.com/knative/community/commit/9cf199f9a0bbbd0b258fdec039c324de19cdc310